samples/nrf9160/agps/src/main.c

```diff
@@ -26,9 +31,57 @@ static uint64_t start_search_timestamp;
 static uint64_t fix_timestamp;
 static struct k_work gps_start_work;
 static struct k_work_delayable reboot_work;
+#if defined(CONFIG_NRF_CLOUD_PGPS)
+static struct k_work_delayable manage_pgps_work;
```

Comment:
This could also be a `k_work` now :) 

I'll approve though, since all `k_work_delayable` items are now resolved.

Reply:
done